compiler/src/dmd/func.d

```diff
@@ -4431,21 +4431,23 @@ extern (C++) final class UnitTestDeclaration : FuncDeclaration
 {
     char* codedoc;      // for documented unittest
     char* name;      // name of unittest
+    uint len;      // length of name
```

Comment:
Either use a const(char)[] OR you could actually just store a StringExp. The latter woudl actually be cheaper.

Reply:
You don't to use char* it's just historical bullshit in the compiler that the other one is there.